infra/run-baseline.rkt

```diff
@@ -76,8 +78,11 @@
        (raise (exn:rival:unsamplable "Unsamplable input" (current-continuation-marks) pt))]
       [else (loop (* 2 prec))])))
 
+(define (baseline-machine-adjust machine)
+  (vector-fill! (baseline-machine-precisions machine) (bf-precision)))
+
 (define (baseline-machine-full machine inputs)
-  ; no adjust here
+  (baseline-machine-adjust machine)
```

Comment:
A little change to baseline. Baseline was kinda fast because it didn't use precision vector (since the precision is uniform anyways), however, for a fair comparison, precision vector has to be added